pkg/karmadactl/cmdinit/karmada/deploy.go

```diff
@@ -4,6 +4,7 @@ import (
 	"context"
 	"encoding/json"
 	"fmt"
+	options2 "github.com/karmada-io/karmada/pkg/karmadactl/options"
```

Comment:
We organize the import package in three parts:
`go package

third-party package

current project package`